### DISCUSSION TITLE
How to make plans for roles? example buyer and seller has different plans?

### DISCUSSION THREAD

**Discussion by jay2success** [bpuig/laravel-subby#157] [Q&A]
Hello,

How to make plans for roles? 
Example: buyer and seller role has different membership plans?

Maybe one of the way is to alter "plan" migration table, add 1 field for roles?
But how do i extend the plan model?

Please help, thanks.

**Comment by bpuig** [comment]
I think you got this right, the time I used the package with roles I added a `role_id` column and then used auth user role to insert and get plans data.

**Comment by jay2success** [comment]
Hi,

Thanks for your answers.
Ok, if i alter the migration table, how do i deal with Plan model? Is there guide to extend the Bpuig Plan model?
Thanks.
